book/src/cluster-driver-registrar.md

```diff
@@ -0,0 +1,29 @@
+# CSI `cluster-driver-registrar`
+
+Git Repository: https://github.com/kubernetes-csi/cluster-driver-registrar
+
+The CSI `cluster-driver-registrar` is a sidecar container that registers a CSI Driver with a Kubernetes cluster by creating a [CSIDriver Object](csi-driver-object.md) which enables the driver to customize how Kubernetes interacts with it.
+
+## Stable Releases
```

Comment:
Should we put the release table here or in the sidecar repo? Putting it in both will almost guarantee that they will be out of sync

Reply:
Yep, I went back and forth on this. It is very important information. We can start with it here, and pull it in to README files per repo later.